radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -172,6 +172,23 @@ elseif(PCB STREQUAL X7)
     add_definitions(-DRADIO_FAMILY_JUMPER_T12)
     add_definitions(-DEEPROM_VARIANT=0x4003)
     add_definitions(-DMANUFACTURER_JUMPER)
+  elseif(PCBREV STREQUAL LR3PRO)
+    set(DEFAULT_INTERNAL_MODULE CROSSFIRE CACHE STRING "Default internal module")
+    set(INTERNAL_MODULE_SERIAL YES)
+    set(FLAVOUR lr3pro)
+    set(NAVIGATION_TYPE 9x)
+    set(CPU_TYPE STM32F2)
+    set(CPU_TYPE_FULL STM32F205xE)  # for size report
+    add_definitions(-DSTM32F205xx)
+    set(LINKER_SCRIPT targets/taranis/stm32f2_flash.ld)
+    set(LUA_EXPORT lua_export_tlite)
+    add_definitions(-DRADIO_TLITE)
```

Comment:
I think one target should only have one RADIO_... define set and now two. So this should be removed and all places where RADIO_TLITE is checked should get a check for RADIO_LR3PRO.
This change also removes the need for the !defined(RADIO_LR3PRO) checks in some of the code files.

Reply:
I think you're right. I've revised the code.

Reply:
Without having it compiled or tested in any way, this looks much better than before.